tests/acceptance/16_cf-serverd/serial/nondefault_ciphers_tlsversion.srv

```diff
@@ -0,0 +1,37 @@
+body common control
+{
+      bundlesequence => { "access_rules" };
+      inputs => { "../../default.cf.sub" };
+
+}
+
+#########################################################
+# Server config
+#########################################################
+
+body server control
+
+{
+      port => "9888";
```

Comment:
On Tue, Jul 14, 2015 at 10:56 AM, Kristian Amlie <notifications@github.com>
wrote:

> Just out of curiosity, if we made all these port numbers unique, could we
> run the server tests in parallel?


Not really, mainly because the same server is sometimes needed by two or
more tests.